docs/Commands.html

```diff
@@ -53,19 +53,19 @@ <h3 id=assemble>Command <code>assemble</code></h3>
 
 <h3 id=cleanupBinaryData>Command <code>cleanupBinaryData</code></h3>
 <p>
-This is used to unmount the filesystem in memory used to
+This command is used to unmount the filesystem in memory used to
 hold Shasta binary data, mounted at the <code>Data</code>
 directory under the assembly directory. 
-This frees the memory (in 4 Kb or 2 MB pages 
+It frees the memory (in 4 Kb or 2 MB pages 
 depending on the setting of <code>--memoryBacking</code>)
 used to store data in the filesystem.
 
 <p>
-This also removes the <code>Data</code> directory.
+It also removes the <code>Data</code> directory.
 If a <code>DataOnDisk</code> directory exists,
-it creates a symbolic link to it, named <code>Data</code>.
-This makes it possible to use Shasta http server functionality
-and the Python API even after the binary data are 
+it creates a symbolic link named <code>Data</code>.
```

Comment:
This new wording does not say what the symbolic link points to.

Reply:
I will leave the original wording